docs/pages/getting-started/docker/index.md

```
@@ -55,10 +55,26 @@ Check the logs:
 You see output like:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)